packages/trace-viewer/index.html

```diff
@@ -26,5 +26,12 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/index.tsx"></script>
+    <dialog id="fallback-error" open="false">
+      <p>The Playwright Trace Viewer must be loaded over the <code>http://</code> or <code>https://</code> protocols.</p>
+    </dialog>
+    <script>
+      if (!/^https?:/.test(window.location.protocol))
+        document.getElementById("fallback-error").open = "true";
+    </script>
```

Comment:
We can probably be more helpful and make suggestions of running either `npx playwright show-report`, `npx playwright show-trace`, using `trace.playwright.dev`, or running/hosting on static server.

Additionally, we could inline the `index.tsx` and re-use the logic/components in there. The current issue is, `index.tsx` is violates CORS policy.